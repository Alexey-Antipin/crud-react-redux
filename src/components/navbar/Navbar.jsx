import React, { useState } from "react";
import { Link } from "react-router-dom";
import "../navbar/Navbar.scss";

export const Navbar = () => {

    const [turnOff, setturnOff] = useState(false)

    const clickHandler = () => {
        setturnOff(!turnOff)
    }

    return (
        <div className="Navbar">
            <header>
                {turnOff == false ?
                    <button
                        className="Navbar__button_menu"
                        onClick={clickHandler}>
                        <Link to={"/"}>В главное меню</Link>
                    </button>
                    :
                    <button
                        className="Navbar__button"
                        onClick={clickHandler}>
                        <Link to={"/PageTodo"}>Список todo</Link>
                    </button>}
            </header>
        </div>
    )
}