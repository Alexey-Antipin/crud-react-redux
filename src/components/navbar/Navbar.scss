.Navbar {
    display: flex;
    justify-content: center;
    background-color: rgb(196, 255, 242);
    .Navbar__button_menu {
        background-color: rgb(161, 248, 229);
        border-radius: 10px;
        border-color: rgba(65, 65, 65, 0.596);
        font-size: 15px;
        padding: 10px;
    }
    .Navbar__button {
        background-color: rgb(161, 248, 229);
        border-radius: 10px;
        border-color: rgba(65, 65, 65, 0.596);
        font-size: 15px;
        padding: 10px 20px;
    }
}