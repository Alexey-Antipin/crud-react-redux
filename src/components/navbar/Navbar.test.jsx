import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Navbar } from "./Navbar";

describe("Navbar", () => {
  it("Navbar spanshot", () => {
    const { asFragment } = render(<Navbar />);
    expect(asFragment(<Navbar />)).toMatchSnapshot();
  });
});
