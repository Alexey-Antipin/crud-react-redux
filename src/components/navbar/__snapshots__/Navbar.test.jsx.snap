// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Navbar spanshot 1`] = `
<DocumentFragment>
  <div
    class="Navbar"
  >
    <header>
      <button
        class="Navbar__button"
      />
    </header>
  </div>
</DocumentFragment>
`;
